xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -254,18 +254,25 @@ std::unique_ptr<PIPEWIRE::CPipewire> pipewire;
 
 bool CAESinkPipewire::Register()
 {
-  pipewire = PIPEWIRE::CPipewire::Create();
-  if (!pipewire)
-    return false;
+  try
+  {
+    pipewire = PIPEWIRE::CPipewire::Create();
+    if (!pipewire)
+      return false;
 
-  AE::AESinkRegEntry entry;
-  entry.sinkName = "PIPEWIRE";
-  entry.createFunc = CAESinkPipewire::Create;
-  entry.enumerateFunc = CAESinkPipewire::EnumerateDevicesEx;
-  entry.cleanupFunc = CAESinkPipewire::Destroy;
-  AE::CAESinkFactory::RegisterSink(entry);
+    AE::AESinkRegEntry entry;
+    entry.sinkName = "PIPEWIRE";
+    entry.createFunc = CAESinkPipewire::Create;
+    entry.enumerateFunc = CAESinkPipewire::EnumerateDevicesEx;
+    entry.cleanupFunc = CAESinkPipewire::Destroy;
+    AE::CAESinkFactory::RegisterSink(entry);
 
-  return true;
+    return true;
+  }
+  catch (const std::exception&)
+  {
+    return false;
```

Comment:
wouldn't it make sense to log the actual exception?

Reply:
I added logging, but unfortunately it doesn't show up in the log because the logger is not yet initialized. But it helps when logging directly to the console :slightly_smiling_face: